.github/workflows/conformance-ingress-shared.yaml

```diff
@@ -141,6 +144,7 @@ jobs:
           # Clean up after sanity check to avoid any conflicts with the conformance test
           kubectl delete -n default -f examples/kubernetes/servicemesh/basic-ingress.yaml
           kubectl delete -n default -f https://raw.githubusercontent.com/istio/istio/release-1.11/samples/bookinfo/platform/kube/bookinfo.yaml
+          sleep ${{ env.DELAY_TIME }}
```

Comment:
Can we probe some state in the cluster to confirm this instead? Sleeps are notorious for either wasting time by taking too long, or then spontaneuosly failing for an unexpected reason if the timeout happens to be exceeded.

Reply:
good point :100: